cmd/call-api-client/main.go

```diff
@@ -99,7 +100,7 @@ func main() {
 	interrupt := make(chan os.Signal, 1)
 	signal.Notify(interrupt, os.Interrupt)
 
-	api_hostport := fmt.Sprintf("%s:%d", cfg.WSServer.Host, cfg.WSServer.Port)
+	api_hostport := fmt.Sprintf("%s:%d", wsServer, cfg.WSServer.Port)
```

Comment:
I agree the idea is good, but I don't like the fact that you dropped whatever was in the config, as it breaks backwards compatibility. Moreover, the port is still taken from the config.
Thus, I would propose a different approach: for the command line argument to overwrite whatever is in the config, and to be used only if it has been provided as an argument. That parameter should also contain the port (as it describes the server).